scdrs/method.py

```diff
@@ -514,7 +551,7 @@ def _correct_background(
             delimiter="\t",
         )
 
-    # Gene-set-wise correction
+    # Second gene set alignment: mean 0
     v_norm_score = v_norm_score - v_norm_score.mean()
     mat_ctrl_norm_score = mat_ctrl_norm_score - mat_ctrl_norm_score.mean(axis=0)
     if save_intermediate is not None:
```

Comment:
save_intermediate may not be necessary anymore because it seems the current implementation can cope with larger data set. But we can defer this to later.

Reply:
save_intermediate is for internal use of generating intermediate scores in Supp. Fig. 1. So we should keep it.